src/BenchmarkDotNet/Templates/CsProj.txt

```diff
@@ -26,6 +26,8 @@
     <AppendTargetFrameworkToOutputPath>true</AppendTargetFrameworkToOutputPath>
     <!-- fix for NETSDK1150: https://docs.microsoft.com/en-us/dotnet/core/compatibility/sdk/5.0/referencing-executable-generates-error -->
     <ValidateExecutableReferencesMatchSelfContained>false</ValidateExecutableReferencesMatchSelfContained>
+    <!-- fix System.Runtime.CompilerServices.Unsafe doesn't support netcoreapp2.0 -->
```

Comment:
nit: In the other project file we have used the word suppress. I am not a native speaker, but to me it feels more like a workaround than a proper fix, so I would stay with suppress.

@timcassell but my understanding is that you are a native speaker, so please feel free to ignore my comment